packages/babel-plugin-transform-react-jsx-development/test/fixtures/linux/handle-fragments-with-key/output.js

```diff
@@ -0,0 +1,8 @@
+var _react = require("react");
+
+var _jsxFileName = "<CWD>/packages/babel-plugin-transform-react-jsx-development/test/fixtures/linux/handle-fragments-with-key/input.js";
+
+var x = _react.jsxDEV(React.Fragment, {}, "foo", false, {
```

Comment:
In this case, because users call `React.Fragment` directly, they would have to import React directly (similar to if they call createElement directly). Let me rewrite the test to reflect this.